### PR TITLE
[18.0][MIG] developer_menu: Migration to 18.0

### PR DESCRIPTION
**Pull request by bealdav**
image

### CODE REVIEW

**Review by florian-dacosta — APPROVED** [review]
LGTM code review

**Review by PaulGoubert — APPROVED** [review]
Functiuonnal review Done. 
Usefull 
LGTM

**Comment by bealdav** [comment]
cc @dreispt

**Review by houssine78 — APPROVED** [review]
LGTM

**Comment by houssine78** [comment]
@pedrobaeza this one is ready to be merge too.

File: checklog-odoo.cfg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[checklog-odoo]
```
Comment (pedrobaeza) [inline checklog-odoo.cfg:1]: Not sure we should disable this at repo level.

**Review by pedrobaeza — COMMENTED** [review]
Please name properly PRs for being found. It's not the first time I tell you about this.

/ocabot migration developer_menu
/ocabot merge nobump

**Comment by bealdav** [comment]
@pedrobaeza Thanks for this merge.

I don't know how to have a better name ?

image

What's wrong ?

**Comment by pedrobaeza** [comment]
That's because I have changed it to the correct one, having the full module name. Please check the log on the conversation to see your old PR title.

**Comment by bealdav** [comment]
Hi sorry, thanks

**Pull request merged**